pkgs/by-name/da/das/package.nix

```
@@ -5,26 +5,28 @@
 
 python3.pkgs.buildPythonApplication rec {
   pname = "das";
-  version = "0.3.8";
-  format = "pyproject";
+  version = "1.0.3";
+  pyproject = true;
 
   src = fetchFromGitHub {
     owner = "snovvcrash";
     repo = "DivideAndScan";
-    rev = "refs/tags/v${version}";
-    hash = "sha256-a9gnEBTvZshw42M/GrpCgjZh6FOzL45aZqGRyeHO0ec=";
+    rev = "v${version}";
```

Revert this. Branch/tags confusion is an issue for the updates made by the bot.

```suggestion
    rev = "refs/tags/v${version}";
```